src/ProductConstructionService/ProductConstructionService.Api/Metrics/TelemetryConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace ProductConstructionService.Api.Metrics;
+
+public static class TelemetryConfiguration
+{
+    public static void AddTelemetry(this WebApplicationBuilder builder)
+    {
+        builder.Services.AddSingleton<ITelemetryRecorder, TelemetryRecorder>();
+        builder.Services.AddApplicationInsightsTelemetry();
+        builder.Services.AddApplicationInsightsTelemetryProcessor<RemoveDefaultPropertiesTelemetryProcessor>();
```

Comment:
Will this remove the fields for all other AI logs or just the custom events?

Reply:
it will remove those two customDimensions everywhere. We could filter for `TelemetryEvent` types if we want to remove those only for events, but maybe we don't need them?

Reply:
Nah, that's fine